lib/src/block_syntaxes/alert_block_syntax.dart

```diff
@@ -25,7 +25,7 @@ class AlertBlockSyntax extends BlockSyntax {
         parser.lines.any((line) => _contentLineRegExp.hasMatch(line.content));
   }
 
-  /// Whether this alert ends with a lazy continuation line.
+  // Whether this alert ends with a lazy continuation line.
```

Comment:
nit: Actually I think all three lines here should use `///`, documenting the member.

Reply:
Personally, if it explains how a method is used rather than implementation details, I still use doc comments for private members as then the analysis server can surface them on hover :)